doc/architecture/adr/0019-message-broker.md

```diff
@@ -0,0 +1,53 @@
+Message broker
+
+* Status:
```

Comment:
Please set Status to "Proposed". Also add "PROPOSAL" to the file name as described here: https://github.com/Energinet-DataHub/energy-origin/blob/main/doc/architecture/adr/process.md. The process description is unfortunately not 100% updated, so please also update the README file with a link to this ADR.